developers/weaviate/modules/retriever-vectorizer-modules/text2vec-huggingface.md

```diff
@@ -14,7 +14,7 @@ The `text2vec-huggingface` module allows you to use [Hugging Face models](https:
 
 * Note: this module uses a third-party API.
 * Note: make sure to check the Inference [pricing page](https://huggingface.co/inference-api#pricing) before vectorizing large amounts of data.
-* Note: Weaviate automatically parallelizes requests to the Inference-API when using the batch endpoint, see the previous note.
+* Note: Weaviate automatically parallelizes requests to the Inference-API when using the batch endpoint.
```

Comment:
"inference API"

Reply:
I will remove the dash; but I think it should be capitalized as it is what their service is called. (https://huggingface.co/inference-api)